packages/vite/src/index.ts

```diff
@@ -167,22 +199,44 @@ export function imagetools(userOptions: Partial<VitePluginOptions> = {}): Plugin
         if (req.url?.startsWith(basePath)) {
           const [, id] = req.url.split(basePath)
 
-          const image = generatedImages.get(id)
+          const { image, metadata } = generatedImages.get(id) ?? {}
 
-          if (!image)
+          if (!metadata)
             throw new Error(`vite-imagetools cannot find image with id "${id}" this is likely an internal error`)
 
+          res.setHeader('Cache-Control', 'max-age=360000')
+
```

Comment:
I know this was already present, but I'm not really sure why we had it. It seems like it just has the potential to cause issues if the underlying image changes. And with the other caching that this plugin has, serving the requests shouldn't be all that expensive. We could potentially add an etag header as an alternative
```suggestion
```

Reply:
I don't think it will cause issues, because every change to the original images or the transform options will change the image ID, too. But we can try without it.